src/Interface/InterfaceView/InterfaceView.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { KeyValue, Col, Row } from '@folio/stripes-components';
```

Comment:
this is wrong import, the purpose to not import from @folio/stripes-components directly. We use import from stripes/components everywhere. And it's specified in docs. Please read the docs if you missed that. https://github.com/folio-org/stripes/blob/master/doc/stripes-framework.md

Reply:
Thank you for useful link, but this line was moved from previous version and I didn't check if it is right import.

Reply:
so if you touch such things - they are marked as green and it's a nice chance to make them better or at least following project common practices